lib/templates.js

```diff
@@ -191,30 +191,32 @@ var templateSpecs = {
           console.log('LESS Generation Error', e);
           return;
         }
-        // With scss enabled this should be appendFileSync
-        fs.writeFileSync(strategy.output,
```

Comment:
Are we sure this was meant to be changed here, and not meant "ofr the scss function use appendFileSync? I see in the diff below you then put writeFileSync, but I can't understand how this comment makes sense. Why does making scss work change how less is built?